docs/changelogs/CHANGELOG-2.22.md

```diff
@@ -0,0 +1,481 @@
+
+#### Supported Versions
+
+- 1.24.8
+- 1.24.9
+- 1.24.10
+- 1.25.4
+- 1.25.6
+- 1.25.6
+- 1.26.1
+
+### Highlights
+
+#### KubeVirt
+
+KubeVirt cloud provider support is leaving the "technical preview" phase and is now considered GA. A [migration guide](https://docs.kubermatic.com/kubermatic/v2.22/architecture/supported-providers/kubevirt/kubevirt/#migration-from-kkp-221-to-kkp-222) is available, please read it before proceeding with the KKP upgrade.
+
+- Fix to ensure that we do not raise an error when reconciling the namespace in the infrastructure KubeVirt cluster, until we get the value of the namespace to create, avoiding transient errors ([#10849](https://github.com/kubermatic/kubermatic/pull/10849))
+- Bugfix for KubeVirt infra CSI token creation due to auto-creation disabled in k8s 1.24 ([#10908](https://github.com/kubermatic/kubermatic/pull/10908))
+- Introduction of KubeVirt default instance types and instance preferences that will replace virtual machine presets ([#11025](https://github.com/kubermatic/kubermatic/pull/11025))
+- Fix wrong quota filtering when `VirtualMachineInstancePreset.spec.cpu` has no quantity but only other fields ([#11046](https://github.com/kubermatic/kubermatic/pull/11046))
+- Add support for ToplogySpreadConstraint for Kubevirt VM ([#11114](https://github.com/kubermatic/kubermatic/pull/11114))
+- Add support for instancetype/preference ([#11182](https://github.com/kubermatic/kubermatic/pull/11182))
+- Update KubeVirt CCM to v0.4.0 and kubevirt.io/api to v0.58.0 ([#11249](https://github.com/kubermatic/kubermatic/pull/11249))
+- Switch StorageClasses init configuration from annotation to DC ([#11716](https://github.com/kubermatic/kubermatic/pull/11716))
+- Update KubeVirt DC: add additional network policies configuration ([#11659](https://github.com/kubermatic/kubermatic/pull/11659))
+- Update seed and cluster to config StorageClasses from seed ([#11701](https://github.com/kubermatic/kubermatic/pull/11701))
+- Graceful user cluster workload eviction in case of bare-metal infra node draining ([#11588](https://github.com/kubermatic/kubermatic/pull/11588))
+- Split CSI driver deployment between user and infra cluster and remove user cluster CSI access to KubeVirt cluster ([#11370](https://github.com/kubermatic/kubermatic/pull/11370))
+- Fix KubeVirt LB issue (wrong custer-isolation netpol): LB not accessible from outside user-cluster ([#11640](https://github.com/kubermatic/kubermatic/pull/11640))
+- Change CustomNetworkPolicies type (extract name) ([#11666](https://github.com/kubermatic/kubermatic/pull/11666))
+- Fix missing ccmClusterName feature in KubeVirt clusters upgraded to KKP 2.22 ([#11844](https://github.com/kubermatic/kubermatic/pull/11844))
+
+#### KubeOne Cluster Support
+
+- Update KubeOne image to v1.5.3 for KubeOne external cluster controller ([#11388](https://github.com/kubermatic/kubermatic/pull/11388))
```

Comment:
Still true after https://github.com/kubermatic/kubermatic/pull/11914 ?

Reply:
No, but the final version to be included isnt merged yet.